### PR TITLE
Adding reconstructed reloc utility sourcecode

### PR DESCRIPTION
**Pull request by leecher1337**
As MSDOS-player compiled version is a bit unreliable, reconstructed reloc.exe utility sourcecode and added to build tools.

### CODE REVIEW

File: buildlocaltools.cmd [hunk 1]
```diff
@@ -124,7 +124,10 @@ call :Build hsplit sdktools\hsplit idw\hsplit.exe idw\hsplit.exe
 if errorlevel 1 goto Error
 
 call :Build wcshdr sdktools\wcshdr idw\wcshdr.exe idw\wcshdr.exe
-if errorlevel 1 goto Error
+if errorlevel 1 goto Error
```
Comment (stephanosio) [inline buildlocaltools.cmd]: Non-change (line ending setting problem)

File: src/placefil.txt [hunk 2]
```diff
@@ -60,7 +60,8 @@ nmake.exe           bldtools
 ntsd.exe            dbg\files\bin
 pcopy.exe           bldtools
 rc.exe              bldtools
-rcdll.dll           bldtools
+rcdll.dll           bldtools
```
Comment (stephanosio) [inline src/placefil.txt]: Non-change (line ending setting problem)

File: src/sdktools/dirs [hunk 3]
```diff
@@ -6,7 +6,8 @@ DIRS= mkmsg       \
       alias       \
       pcopy       \
       rcdll       \
-      rc          \
+      rc          \
```
Comment (stephanosio) [inline src/sdktools/dirs]: Non-change (line ending setting problem)

File: src/sdktools/dirs [hunk 4]
```diff
@@ -6,7 +6,8 @@ DIRS= mkmsg       \
       alias       \
       pcopy       \
       rcdll       \
-      rc          \
+      rc          \
+	  reloc       \
```
Comment (stephanosio) [inline src/sdktools/dirs]: Please use SPACEs

File: src/sdktools/reloc/reloc.c [hunk 5]
```diff
@@ -0,0 +1,148 @@
+/***
```
Comment (stephanosio) [inline src/sdktools/reloc/reloc.c]: The entire file uses CRLF line ending. Please use LF.

**Comment by leecher1337** [comment]
Sorry, no idea how to amend a SVN commit, so I just pushed to requested CRLF changes to my repository.

**Pull request merged**